### DISCUSSION TITLE
Files are non-identical after sequentially reading

### DISCUSSION THREAD

**Discussion by Guhlsev** [keirf/greaseweazle#452] [Q&A]
I use "blue pill" board
I suspect my STM board is a fake. I read disk several times (in my case just MS-DOS loading disk) and get non-identical files.
I even made "erase" command and rad after that, but result is the same - non-identical files.
I'd like to know, if I read one disk two or more times serially, should I get the same files (size, binary)?
I used gw read mydisk.scp
If I write mydisk.scp onto disk, th PC doesn't see it.

**Comment by keirf** [comment]
SCP image files will never match. They are raw flux images and depend on exact flux timings (which jitter) and index timings.

**Comment by keirf** [comment]
Try reading to an IMG file using a format like ibm.1440: gw read --format=ibm.1440 name.img

**Comment by Guhlsev** [comment]
Thanks for response 
I will try
Now I use MS-DOS loading disk, because I can test this method easily on my PC
Then I'm going to make flux image of special Mitsubishi floppy disc, which is used in Mitsubishi CNC

**Reply by keirf** [reply]
For that one you may need to record raw flux in an SCP image. Or perhaps something like:
```gw read --format=ibm.scan name.imd```
